purchase_all_shipments/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": "Purchase All Shipments",
+    "version": "11.0.1.0.0",
+    "author": "Camptocamp,"
+              "Odoo Community Association (OCA),",
+    "category": "Purchases",
+    "license": "AGPL-3",
+    "depends": [
+        "purchase",
+    ],
+    "data": [
+        "view/purchase_order.xml",
+    ],
+}
```

Comment:
Website key is missing and it would be great if you can add a summary as well.

Reply:
updated